spec/unit/indirector/catalog/compiler_spec.rb

```diff
@@ -150,15 +150,9 @@ def a_request_that_contains(facts)
       @compiler.extract_facts_from_request(request).should be_nil
     end
 
-    it "deserializes the facts and timestamps them" do
-      @facts.timestamp = Time.parse('2010-11-01')
+    it "should deserialize the facts" do
       request = a_request_that_contains(@facts)
-      now = Time.parse('2010-11-02')
-      Time.stubs(:now).returns(now)
-
       facts = @compiler.extract_facts_from_request(request)
-
-      facts.timestamp.should == now
```

Comment:
It doesn't look like we are "expecting" anything? Perhaps we should set an explicit timestamp on the facts, generate the request, extract the facts, and verify the timestamp is the same?

<pre>
time = Time.now
@facts.timestamp = time
request = a_request_that_contains(@facts)
facts = @compiler.extract_facts_from_request(request)
expect(facts.timestamp).to eq(time)
</pre>